documentation/c12-tricks.sil

```diff
@@ -161,10 +161,10 @@ function diglot:endPage = ()
 end
 \end{verbatim}
 
-Similarly for the end of the document, but in this case we will use the
-emergency \code{chuck} method; whereas \code{leaveHmode} means “call the
-page builder and see there’s enough material to build a page”, \code{chuck}
-means “you must get rid of everything on your queue now.” We add some infinitely
+At the end of the document, we will use the
+emergency \code{chuck} method. Where \code{leaveHmode} means “call the
+page builder and see there’s enough material to build a page,” \code{chuck}
+means “you must get rid of everything on your queue \em{now.”} We add some infinitely
```

Comment:
Not sure the quotation mark should be in italic. (Some typographers would perhaps frown: the quote end and start mark should be in the same style, most likely.)

Reply:
Going to have to side with the French on this one ;-)